mmengine/optim/optimizer/optimizer_wrapper.py

```diff
@@ -295,10 +329,16 @@ def get_lr(self) -> Dict[str, List[float]]:
         Provide unified interface to get learning rate of optimizer.
 
         Returns:
-            Dict[str, List[float]]: Learning rate of the optimizer.
+            Dict[str, List[float]]:param_groups Learning rate of the optimizer.
         """
-        lr = [group['lr'] for group in self.param_groups]
-        return dict(lr=lr)
+        res = defaultdict(list)
+        if self.base_param_settings is not None:
+            res['base_lr'].append(self.base_param_settings['lr'])
+
+        for group in self.param_groups:
+            res['lr'].append(group['lr'])
+
+        return dict(res)
```

Comment:
```suggestion
        res = {}
        if self.base_param_settings is not None:
            res['base_lr'] = [self.base_param_settings['lr']]

        res['lr'] = [group['lr'] for group in self.param_groups]:
        return res
```

Reply:
Should we replace the `self.param_groups` with `self.optimizer.param_groups`?

Reply:
I think we could use `self.optimizer.param_groups` because the tracker parameter is exclusive .